identity-wallet/src/state/mod.rs

```diff
@@ -138,69 +131,69 @@ impl AppState {
     }
 }
 
-#[cfg(test)]
-mod tests {
-    use super::*;
-    use crate::state::profile_settings::Locale;
-    use crate::state::profile_settings::Profile;
-    use indoc::indoc;
-    use tests::profile_settings::AppTheme;
+// #[cfg(test)]
```

Comment:
We should either make the test run again or remove it entirely, but not comment it out without further notice or comment _why_ it was commented.

Reply:
updated